src/components/FeeSelector/shared.tsx

```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { SupportedChainId } from 'constants/chains'
+import { ReactNode } from 'react'
+
+export const FEE_AMOUNT_DETAIL: Record<
+  FeeAmount,
+  { label: string; description: ReactNode; supportedChains: SupportedChainId[] | 'all' }
+> = {
+  [FeeAmount.LOWEST]: {
+    label: '0.01',
+    description: <Trans>Best for very stable pairs.</Trans>,
+    supportedChains: [SupportedChainId.MAINNET],
+  },
+  [FeeAmount.LOW]: {
+    label: '0.05',
+    description: <Trans>Best for stable pairs.</Trans>,
+    supportedChains: 'all',
```

Comment:
ALL_SUPPORTED_CHAIN_IDS?

Reply:
or just `allowedChains: undefined | SupportedChainId[]`

Reply:
one thing to consider is tick spacing may differ between the fee amounts on different chains. not sure where tick spacing is coming from here, is it being queried from the factory?

Reply:
TICK_SPACINGS is updated here: https://github.com/Uniswap/v3-sdk/blob/main/src/constants.ts#L25

Reply:
that constant already exists in the chains constants file

i'm also fine with `'all'`

Reply:
oh nice, i prefer `ALL_SUPPORTED_CHAIN_IDS` if it's already defined.